presto-docs/src/main/sphinx/release/release-0.140.rst

```diff
@@ -12,11 +12,14 @@ General Changes
 * Fix race which can cause queries to fail with a ``REMOTE_TASK_ERROR``.
 * Optimize :func:`array_distinct` for ``array<bigint>``.
 * Optimize ``>`` operator for :ref:`array_type`.
+* Fix projection push down to not duplicate non-deterministic functions
+* Fix predicate move around to not duplicate columns
```

Comment:
Ah, so this is the performance issue... Then I'd rephrase as:
* Improve performance of joins, aggregations, etc. by removing unnecessarily duplicated columns.